generators/CoreCLRStatefulActor/index.js

```diff
@@ -95,16 +95,36 @@ var ClassGenerator = generators.Base.extend({
     var codePath = path.join(appPackage , appPackagePath, servicePackage, 'Code');
     var testCodePath = path.join(appPackage , serviceProjName + 'TestClient' );
 
+    var is_Windows = (process.platform=='win32');
+    var is_Linux = (process.platform=='linux');
+    var is_mac = (process.platform=='darwin');
+
+    var extension1;
```

Comment:
Better to name this as sdkScriptExtension and entryPointScriptExtension

Reply:
Done named them as sdkScriptExtension and buildScriptExtension as we are not using entrypoint script for windows any more.